code/modules/photography/camera/camera.dm

```diff
@@ -158,6 +168,8 @@
 	qdel(P)
 
 /obj/item/camera/proc/captureimage(atom/target, mob/user, flag, size_x = 1, size_y = 1)
+	if(!istype(user)) //this runtimes because it assumes user has a client and is the correct type
```

Comment:
If we are passing in any atom as a user, then it should be typed as `atom/user`.